app/views/shopping_lists/_form.html.haml

```diff
@@ -0,0 +1,16 @@
+#form-container.bg-light.mx-5
+  = form_for @shopping_list do |f|
+    .row.m-2.py-4
+      .col-6.col-sm-2
+        #inputShoppingListStatus= f.select :status, options_for_select(ShoppingList.statuses.keys), {}, class: "form-select"
```

Comment:
I would move `ShoppingList.statuses.keys` to controller or helper so it's not called directly from view